gallery/src/pages/components/ha-selector.ts

```diff
@@ -73,6 +73,7 @@ const SCHEMAS: {
         selector: { select: { options: ["Option 1", "Option 2"] } },
       },
       icon: { name: "Icon", selector: { icon: {} } },
+      media: { name: "Media", selector: { media: {} } },
```

Comment:
Can we also add it to the ha-form gallery page. It has a form with all selectors.

Reply:
Also, we have automation/selectors.ts too. We have 2 design pages for selectors? 🤦 

https://620d65c9f5140e8acf3cf643--home-assistant-gallery.netlify.app/#components/ha-selector

Changing between browse and manual logs a JS error.